web/crux/proto/common.proto

```diff
@@ -137,6 +137,7 @@ message Routing {
   optional string path = 101;
   optional bool stripPath = 102;
   optional string uploadLimit = 103;
+  optional uint32 port = 104;
```

Comment:
why not uint16?

Reply:
Not available in proto, or it was bugging.